src/jit/compiler.cpp

```diff
@@ -919,15 +919,6 @@ var_types Compiler::getArgTypeForStruct(CORINFO_CLASS_HANDLE clsHnd,
         *wbPassStruct = howToPassStruct;
     }
 
-#if defined(FEATURE_MULTIREG_ARGS) && defined(_TARGET_ARM64_)
-    // Normalize struct return type for ARM64 FEATURE_MULTIREG_ARGS
-    // This is not yet enabled for ARM32 since FEATURE_SIMD is not enabled
-    if (varTypeIsStruct(useType))
-    {
-        useType = impNormStructType(clsHnd);
-    }
-#endif
-
```

Comment:
This code was added incorrectly.  To fix a problem with SIMD16 multireg arg/return.

It is problematic because it makes it impossible to distinguish between SIMD16 passed by Value, and SIMD16 passed by primitive type.  It breaks the contract of these two functions.